src/content/templates/graph-network_summary.json

```diff
@@ -0,0 +1,222 @@
+{
```

Comment:
See (B). We can still merge it into graph-if, by extending the variable graph feature a little more.